ApiHelper.php

```diff
@@ -293,7 +293,7 @@ public function setText( $page, $text, $section = false ) {
 		$params = [
 			'title' => $page,
 			'text' => $text,
-			'summary' => 'Popular pages report update',
+			'summary' => $summary ? $summary:'Popular pages report update',
```

Comment:
Could be simplified to `$summary ?? 'Popular pages report update'`

Reply:
done